doc/coding-conventions.xml

```diff
@@ -244,30 +248,29 @@ bound to the variable name <varname>e2fsprogs</varname> in
   <listitem><para>Generally, try to stick to the upstream package
   name.</para></listitem>
 
-  <listitem><para>Don’t use uppercase letters in the
-  <literal>name</literal> attribute — e.g.,
-  <literal>"mplayer-1.0rc2"</literal> instead of
-  <literal>"MPlayer-1.0rc2"</literal>.</para></listitem>
+  <listitem><para>The <literal>name</literal> attribute MUST not contain
+  uppercase letters — e.g., <literal>"mplayer-1.0rc2"</literal> instead
+  of <literal>"MPlayer-1.0rc2"</literal>.</para></listitem>
 
   <listitem><para>The version part of the <literal>name</literal>
-  attribute <emphasis>must</emphasis> start with a digit (following a
-  dash) — e.g., <literal>"hello-0.3.1rc2"</literal>.</para></listitem>
+  attribute MUST start with a digit (following a dash) — e.g.,
+  <literal>"hello-0.3.1rc2"</literal>.</para></listitem>
 
   <listitem><para>If a package is not a release but a commit from a repository, then
-  the version part of the name <emphasis>must</emphasis> be the date of that 
-  (fetched) commit. The date must be in <literal>"YYYY-MM-DD"</literal> format.
+  the version part of the name MUST be the date of that 
+  (fetched) commit. The date MUST be in <literal>"YYYY-MM-DD"</literal> format.
   Also append <literal>"unstable"</literal> to the name - e.g.,
   <literal>"pkgname-unstable-2014-09-23"</literal>.</para></listitem>
 
-  <listitem><para>Dashes in the package name should be preserved
+  <listitem><para>Dashes in the package name SHOULD be preserved
   in new variable names, rather than converted to underscores
   (which was convention up to around 2013 and most names
    still have underscores instead of dashes) — e.g.,
   <varname>http-parser</varname> instead of
   <varname>http_parser</varname>.</para></listitem>
 
   <listitem><para>If there are multiple versions of a package, this
-  should be reflected in the variable names in
+  SHOULD be reflected in the variable names in
   <filename>all-packages.nix</filename>,
   e.g. <varname>json-c-0-9</varname> and <varname>json-c-0-11</varname>.
   If there is an obvious “default” version, make an attribute like
```

Comment:
I think this can be more sharply stated as "if there is a version used by the majority/supermajority of the packages, make an attribute like...".